config/development.toml

```diff
@@ -717,3 +717,16 @@ encryption_key = "A8EF32E029BC3342E54BF2E172A4D7AA43E8EF9D2C3A624A9F04E2EF79DC69
 
 [locker_based_open_banking_connectors]
 connector_list = ""
+
+[network_tokenization_supported_card_networks]
+card_networks = "Visa"
```

Comment:
Some of the config files have `visa`, while `development.toml` and `docker_compose.toml` have `Visa`. Which of these two is the correct value?